app/controllers/days_controller.rb

```diff
@@ -7,16 +7,9 @@ def new
   end
 
   def create
-    current_day = Day.current_for(current_instructor.current_cohort)
     day = Day.new(day_params)
     day.cohort = current_instructor.current_cohort
     if day.save
-      current_day.unaccounted_for_students.each do |student|
```

Comment:
I'm re-working that right now. I'm going to define absence as a lack of a checkin for a student on a certain day. This will make it so we can have a report calculate all it's metrics based on a inputted day.